<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCamera</class>
 <widget class="QDialog" name="AddCamera">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit/Create Camera</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QLabel" name="label_Longitude">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="cameraName"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="cameraLatitude"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="cameraOrientation"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="cameraLongitude"/>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="cameraAltitude"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_Required">
        <property name="text">
         <string>(required)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_cameraAltitude">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Latitude">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_CameraOrientation">
        <property name="text">
         <string>Orientation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_CameraName">
        <property name="text">
         <string>Camera Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_descName">
        <property name="text">
         <string>Descriptive Name</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QLabel" name="label_AdditionalInfo">
        <property name="text">
         <string>Additional Information</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="6">
       <widget class="QPlainTextEdit" name="additionalInfo"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="cameraDescName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>258</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cameraName</tabstop>
  <tabstop>cameraDescName</tabstop>
  <tabstop>cameraLatitude</tabstop>
  <tabstop>cameraLongitude</tabstop>
  <tabstop>cameraOrientation</tabstop>
  <tabstop>cameraAltitude</tabstop>
  <tabstop>additionalInfo</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
